content/04.results.md

```diff
@@ -41,3 +41,18 @@ We observed that including one label function of a mismatched type to distant su
 We found that usually adding random noise did not improve performance (Figure {@fig:auroc_random_label_function_performance} and Supplemental Figure {@fig:aupr_random_label_function_performance}).
 For the CbG edge type we did observe slightly increased performance via AUPR (Supplemental Figure {@fig:aupr_random_label_function_performance}).
 However, performance changes in general were smaller than those observed with mismatched label types.
+
+### Reconstructing Hetionet
+
+![
+Barchat that quantifies how many existing edges we could recall and novel edges that can be added to hetionet.
+Existing edges are shown in the green bar and the blue bar represents novel edges can be added.
+](https://raw.githubusercontent.com/danich1/snorkeling/86037d185a299a1f6dd4dd68605073849c72af6f/figures/edge_prediction_experiment/edges_added.png){#fig:hetionet_reconstruction}
+
+Using a cutoff value of 0.5, we evaluated how many edges we can recall/add to Hetionet v1 (Figure {@fig:hetionet_reconstruction} and Supplemental Table {@tbl:edge_prediction_tbl}).
```

Comment:
I would probably start this with the recall stuff that's below. Tell folks how many of things we already knew get recovered. Then talk about how many edges one could add (at some threshold - why this threshold?). You are diving in pretty deep here right off the bat. Help bring folks into the same understanding that you have so that they can get insight from your paper.